containers/HomeHeaderRight/HomeHeaderRight.js

```diff
@@ -5,6 +5,7 @@ import React, { PureComponent } from 'react';
 import { View, Text, StyleSheet, TouchableOpacity } from 'react-native';
 import { connect } from 'react-redux';
 import { showAccountIssues } from 'actions';
+import I18n from 'utils';
```

Comment:
@ovr It does not work and I absolutely do not understand WHY, please, explain what is wrong here.
I tried it too `import I18n from 'utils/i18n'`, but it's all to no avail, it's probably very stupid, it's elementary things, but damn, I do not know :cry: